vars/buildPlugin.groovy

```diff
@@ -89,6 +86,7 @@ def call(Map params = [:]) {
                                         '--update-snapshots',
                                         "-Dmaven.repo.local=$m2repo",
                                         '-Dmaven.test.failure.ignore',
+                                        '-Dspotbugs.failOnError=false'
```

Comment:
should we added the checkstyle vars here so the build doesn't fail but gets marked as unstable?
```suggestion
                                        '-Dspotbugs.failOnError=false',
                                        '-Dcheckstyle.failOnViolation=false',
                                        '-Dcheckstyle.failsOnError=false'
```